docs/configuration.md

```diff
@@ -120,7 +120,8 @@ of the most common options to set are:
   <td><code>spark.driver.cores</code></td>
   <td>1</td>
   <td>
-    Number of cores to use for the driver process, only in cluster mode.
+    Number of cores to use for the driver process, only in cluster mode. This can be set through
+    <code>--driver-cores</code> command line option.
```

Comment:
I don't think the purpose of this file is to document how the CLI works. It should stick to documenting the underlying properties.